pkg/providers/amifamily/types.go

```diff
@@ -29,17 +29,12 @@ import (
 	v1 "github.com/aws/karpenter-provider-aws/pkg/apis/v1"
 )
 
-const (
-	// AMIVersionLatest is the version used in EKS aliases to represent the latest version. This maps to different
-	// values in the SSM path, depending on the AMI type (e.g. "recommended" for AL2/AL2023)).
-	AMIVersionLatest = "latest"
-)
-
 type AMI struct {
 	Name         string
 	AmiID        string
 	CreationDate string
 	Requirements scheduling.Requirements
+	Deprecated   bool
```

Comment:
Why is this just getting added now? I thought that we already had tracking for deprecations on this version of Karpenter?

Reply:
Not on v1.0.x, no.